test cases/common/56 custom target/meson.build

```diff
@@ -1,6 +1,9 @@
 project('custom target', 'c')
 
-python = find_program('python3')
+python = find_program('python3', required : false)
```

Comment:
Should probably use the python3 module here since we don't use `find_program` on a script.